builder-image/image-jdk21-overrides.yaml

```diff
@@ -0,0 +1,28 @@
+schema_version: 1
+
+name: &imgName "jboss-eap-8/eap8-openjdk21-builder-openshift-rhel8"
+labels:
+    - name: "org.jboss.product"
+      value: "eap8-openjdk21-builder"
+    - name: "com.redhat.component"
+      value: "jboss-eap8-openjdk21-builder-openshift-container"
+    - name: name
+      value: *imgName
+
+envs:
+    - name: IMAGE_NAME
+      value: *imgName
+
+modules:
+      repositories:
+          - name: openjdk
+            git:
+              url: https://github.com/rh-openjdk/redhat-openjdk-containers
+              ref: wildfly-container-30.0
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "21"
+          - name: jboss.container.maven
+            version: "3.8.17"
+          # Required to remove JDK17
```

Comment:
is that also relevant for JDK 21?

Reply:
Maven 3.8.17 brings JDK17 that we remove with the singleton module. This is what openjdk team has done for their first ubi8 jdk21 images: https://github.com/rh-openjdk/redhat-openjdk-containers/blob/wildfly-container-30.0/ubi8-openjdk-21.yaml#L52